sacrebleu.py

```diff
@@ -1296,9 +1296,9 @@ def compute_bleu(correct: List[int],
     smooth_mteval = 1.
     effective_order = NGRAM_ORDER
     for n in range(NGRAM_ORDER):
-        if smooth_method == 'add-k' and n > 1:
```

Comment:
I agree with this bug fix. I suggest to add a comment about 0-based n indexing (but 1-based `NGRAM_ORDER` and `effective_order`).
Alternatively, we could switch to 1-based indexing, i.e. keep `n > 1`, but use `for n in range(1, NGRAM_ORDER+1)` and change all occurrences of indexing by `n` into `n-1`.

Reply:
I checked the code and saw that the second variant is how it's also done in other locations, e.g. line 1413:
```        for ngram in sys_ngrams.keys():
            n = len(ngram.split())
            correct[n-1] += min(sys_ngrams[ngram], ref_ngrams.get(ngram, 0))
            total[n-1] += sys_ngrams[ngram]```
so I'll change it to 1-based indexing.